en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -1,193 +1,212 @@
+!!! note "Important"
+    Enabling this feature and mapping email domains to organizations is only permitted in the root organization. 
 
-### Using the Console
+=== "Using the Console"
 
-1. Login to the organization (root) from the {{ product_name }} Console.
-2. On the {{ product_name }} Console, go to **Login & Registration**, and click **Organization Discovery** under **Organization Settings**.
-3. Turn on the toggle to enable email domain based organization discovery.
+    !!! note "Before you begin"
 
-    !!! note
-        When this is enabled, following restrctions will apply to child organizations during federated authentication and user onboarding.
+        Ensure you have configured email address as a username option. Learn more about [multi-attribute login]({{base_path}}/guides/authentication/multi-attribute-login/).
 
-        - Users can self-register, and administrators can onboard users to child organizations, only if the users' email domains match the domains mapped to the corresponding child organization.
+    To do so,
 
-        {% if (product_name == "WSO2 Identity Server") %}
+    1. Login to the root organization Console.
+    2. Navigate to **Login & Registration**, and under **Organization Settings**, click **Organization Discovery**.
+    3. Turn the toggle on to enable email domain based organization discovery.
 
-        - Federated authentication and Just-In-Time (JIT) provisioning for child organizations are restricted to email domains mapped to those child organizations.
+        ![Enable email domain based organization discovery]({{base_path}}/assets/img/guides/organization/manage-organizations/enable-email-domain-based-organization-discovery.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
 
-        {% endif %}
-{% if (product_name == "WSO2 Identity Server") %}
+    {% if (product_name == "WSO2 Identity Server") %}
 
-4. Select the **Email domain discovery for self-registration** checkbox if you want to allow users to discover and self-register in child organizations based on their email domain.
+    4. Select the **Email domain discovery for self-registration** checkbox if you want to allow users to discover child organizations based on their email domain and self-register in them.
 
-    !!! note
-        To use this capability, self-registration must be enabled in the child organizations. Currently, enabling self-registration for child organizations via the console is not supported. Instead, you need to add the following configuration to the `deployment.toml` file located in the `<IS_HOME>/repository/conf` directory to enable self-registration server-wide.
+        !!! note
+            For this to work, child organizations should have self-registration enabled. Currently, you can enable self-registration for all child organizations by adding the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.
```

Comment:
```suggestion
            For this to work, child organizations should have self-registration enabled. Currently, you can enable self-registration for all organizations by adding the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.
```

Reply:
fixed with 3033eb4d1d38180c4ca1d0ad42b202f75b46363f

Reply:
For IS 7.0.0

<img width="776" alt="image" src="https://github.com/user-attachments/assets/d427b4ef-02ca-4445-a444-07b3d94b4d90" />

For next

<img width="776" alt="image" src="https://github.com/user-attachments/assets/6879526f-efa6-4b25-bde2-4a4f92ce6de3" />